data/json/recipes/other/cords_and_ropes.json

```
@@ -457,7 +457,6 @@
     "subcategory": "CSC_OTHER_MATERIALS",
     "skill_used": "tailor",
     "time": "1 m",
```

I think this was the intention, seperate one 30' rope into 5 6' ropes.
```suggestion
    "time": "1 m",
    "byproducts": [ [ "rope_6", 4 ] ],
```

---

I accidentally changed the wrong recipe. See https://github.com/CleverRaven/Cataclysm-DDA/pull/54319/commits/0e1c1c7812070878df89ccd7739a863d85d5555d.